easybuild/easyconfigs/f/freetype/freetype-2.8.1-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,22 @@
+# Contributed by Luca Marsella (CSCS)
+name = 'freetype'
+version = "2.8.1"
+
+homepage = 'http://freetype.org'
+description = """FreeType 2 is a software font engine that is designed to be small, efficient, highly customizable, and
```

Comment:
```
description = """FreeType 2 is a software font engine that is designed to be
small, efficient, highly customizable, and portable while capable of producing
high-quality output (glyph images). It can be used in graphics libraries,
display servers, font conversion tools, text image generation tools, and many
other products as well."""
```

Reply:
?